### ISSUE TITLE
Output Log Path using host paths

### ISSUE THREAD

**Issue by kellrott** [ohsu-comp-bio/funnel#290]
Looking at the `logs.outputs.path` it looks like the path from the host VM is being recorded, rather then the docker container path. Thus it looks like `/mnt/funnel/b7to4ir3qt7gv3763k7g/var/run/cwl/gaf.json`

Example
```
"logs" : [
      {
         "outputs" : [
            {
               "path" : "/mnt/funnel/b7to4ir3qt7gv3763k7g/var/run/cwl/gaf.json",
               "size_bytes" : "70989749",
               "url" : "swift://biostream/job1/output/gaf.json"
            }
         ],
         "logs" : [
            {
               "host_ip" : "10.50.50.55",
               "start_time" : "2017-11-02T20:38:46.032906942Z",
               "end_time" : "2017-11-02T20:39:28.485524782Z"
            }
         ],
         "end_time" : "2017-11-02T20:39:30.815287726Z",
         "start_time" : "2017-11-02T20:38:44.187495026Z"
      }
   ],
```

**Comment by kmhernan** [comment]
Hmm I don't think I'm seeing this. I have been using this version @buchanae 

```
git commit: cc87fda
git branch: master
git upstream: git@github.com:ohsu-comp-bio/funnel.git
build date: Thu Nov  2 10:29:41 PDT 2017
version: 0.3.0
```

**Comment by buchanae** [comment]
The fix isn't in 0.3.0. We're release 0.4 today, or you can get the latest with `go get github.com/ohsu-comp-bio/funnel`. We'll also try to be better about noting when a fix will be released.
